certora/specs/CER-83-Set.spec

```diff
@@ -0,0 +1,180 @@
+
```

Comment:
it seems that this file will be used to prove many CER-83 related properties. can we at least have inline comments of what gets proven where? i.e. `// CER-86 Set contains return`

Reply:
Good idea. I left comments to link the individual rules to linear issues just now.